src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -420,6 +425,8 @@ Emscripten_HandleTouch(int eventType, const EmscriptenTouchEvent *touchEvent, vo
     xscale = window_data->window->w / client_w;
     yscale = window_data->window->h / client_h;
 
+    bool preventDefault = false;
```

Comment:
Right, I forgot C doesn't have booleans. Would `SDL_bool` really make more sense here though, as we have return type int and return this variable in the end?